library/ssl_misc.h

```diff
@@ -89,6 +91,8 @@
 
 #if defined(MBEDTLS_SSL_PROTO_TLS1_2)
 #define MBEDTLS_SSL_MAX_MINOR_VERSION           MBEDTLS_SSL_MINOR_VERSION_3
+#elif defined(MBEDTLS_SSL_PROTO_TLS1_3)
```

Comment:
This is not correct if both MBEDTLS_SSL_PROTO_TLS1_2 and MBEDTLS_SSL_PROTO_TLS1_3 are defined. MBEDTLS_SSL_MAX_MINOR_VERSION is then equal to MBEDTLS_SSL_MINOR_VERSION_3 while it should be equal to MBEDTLS_SSL_MINOR_VERSION_4.

Reply:
It is relative with hybrid mode. I fix it with this change.

I agree with you. it has been changed. To fix hybrid issues , I put the code in same commit. Please check.